addons/mail/static/src/models/messaging/messaging.js

```diff
@@ -103,7 +103,7 @@ function factory(dependencies) {
             if (this.env.messaging.device.isMobile) {
                 // messaging menu has a higher z-index than views so it must
                 // be closed to ensure the visibility of the view
-                this.env.messaging.messagingMenu.close();
+                this.env.messaging.chatWindowManager.closeAll();
```

Comment:
Ok to add a close on all chat windows here.

But the close on messaging menu was useless if you remove it?

Also the comment above needs to be updated.

Reply:
I was change comment. Can you review again?